src/tasks/WasmAppBuilder/WasmAppBuilder.cs

```diff
@@ -23,6 +23,8 @@ public class WasmAppBuilder : Task
     [Required]
     public string? MainJS { get; set; }
 
+    public string? WasmEnableES6 { get; set; }
```

Comment:
```suggestion
    public bool WasmEnableES6 { get; set; }
```

Reply:
It didn't work for me., will leave that battle to you :)